apps/end-tenancy/views/partials/items-table.html

```diff
@@ -17,7 +17,7 @@
           {{#fields}}
             <td>
               {{value}}
-              <a href="{{baseUrl}}/{{route}}/{{field}}/{{id}}/change" class="edit-link">{{editText}}</a>
+              <a href="{{baseUrl}}/{{route}}/{{subroute}}/{{id}}/change" class="edit-link">{{editText}}</a>
```

Comment:
in this case yeah, as this partial is only used with the loop controller, which defines each local